### DISCUSSION TITLE
Support querying Users (GET) to support analytics pipeline solutions

### DISCUSSION THREAD

**Discussion by Tjmstudios** [langfuse/langfuse#2176] [Ideas]
### Describe the feature or potential improvement

Feature Request: API Endpoint for User Data Export
Description:

I need the ability to export user data from Langfuse for integration with existing enterprise reporting tools. This requires a dedicated API endpoint.

Minimum Required Fields:

userId
firstObservation (timestamp)
lastObservation (timestamp)

Nice-to-Have Fields:
totalObservations
totalTraces
promptTokens
completionTokens
totalTokens
Justification:

Exporting user data via the API is crucial for incorporating Langfuse insights into our broader data analysis workflows and reporting dashboards.

**Comment by marcklingen** [comment]
Users as in userId on traces? Great idea

**Reply by Tjmstudios** [reply]
Yes, the same userId on traces.
